### ISSUE TITLE
emmeans::CLD is deprecated

### ISSUE THREAD

**Issue by rvlenth** [aravind-j/augmentedRCBD#5]
For some time, I have had `emmeans::CLD` as a deprecated function. With the latest version, I removed it completely, but that caused a check error with your package. 
```
Package: augmentedRCBD
Check: Rd cross-references
New result: WARNING
  Missing link or links in documentation object 'augmentedRCBD.Rd':
    ‘[emmeans:CLD.emmGrid]{cld.emmGrid}’
```
This actually is a result of my moving the location of the documentation. I moved it back, and the error goes away. With a little luck, this will not become an issue when I subsequently remove the function altogether. But please make sure you tell your users to use `multcomp::cld()` -- NOT `CLD()` -- if they want these displays.

**Comment by aravind-j** [comment]
Thanks for the heads up. I guess the link should be now `‘[emmeans]{cld.emmGrid}’.`

**Comment by aravind-j** [comment]
Closed in [21a1dbb](https://github.com/aravind-j/augmentedRCBD/commit/21a1dbb1ae44333bbf1b1f3004c41323b346f00a)
